pkg/cmd/status.go

```diff
@@ -65,81 +66,130 @@ func getData(objType string, context string, namespace string, tenant string) (s
 		if err != nil {
 			return "", errors.Wrap(err, "error getting deployment info")
 		}
+	} else {
+		objectData, err := TempFS.ReadFile("templates/show_postgresdump.tpl")
+		if err != nil {
+			return "", errors.Wrap(err, "error reading show_postgresdump.tpl")
+		}
+		data, err = osExec.Command("kubectl", "get", "postgresdumps.db.controllers.ridecell.io", "-n", namespace, "--context", context, "-o", "go-template="+string(objectData)).Output()
+		if err != nil {
+			return "", errors.Wrap(err, "error getting postgresdump instance info")
+		}
 	}
 
 	return string(data), err
 }
 
 var statusCmd = &cobra.Command{
-	Use:   "status [follow] <cluster_name>",
+	Use:   "status",
 	Short: "Get status report of an Summon Instance",
 	Long:  "Shows status details for all components of a Summon Instance",
 	Args: func(_ *cobra.Command, args []string) error {
-		if len(args) == 0 {
-			return fmt.Errorf("cluster name argument is required")
-		}
-		if len(args) > 1 {
-			return fmt.Errorf("too many arguments")
-		}
 		return nil
 	},
 	PreRunE: func(cmd *cobra.Command, args []string) error {
-
 		utils.CheckVPN()
 		utils.CheckKubectl()
 		return nil
 	},
-	RunE: func(_ *cobra.Command, args []string) error {
+	RunE: func(cmd *cobra.Command, args []string) error {
+		statusTypes := []string{"Summon Platform Status", "DB Backup Status"}
```

Comment:
No need to have `Status` here as the command itself is for status. only summon platform and db backup is enough